XTemplate/xtpl.php

```diff
@@ -374,37 +405,48 @@ function out ($bname) {
 	echo $this->text($bname);
 }
 
-/***[ reset ]***************************************************************/
-/*
-	resets the parsed text
-*/
+    /***[ reset ]**************************************************************
+     * @param $bname
+     */
+    /*
+        resets the parsed text
+    */
 
 function reset ($bname) {
 	$this->parsed_blocks[$bname]="";
 }
 
-/***[ parsed ]**************************************************************/
-/*
-	returns true if block was parsed, false if not
-*/
+    /***[ parsed ]*************************************************************
+     * @param $bname
+     * @return bool
+     */
+    /*
+        returns true if block was parsed, false if not
+    */
 
 function parsed ($bname) {
 	return (!empty($this->parsed_blocks[$bname]));
 }
 
-/***[ SetNullString ]*******************************************************/
-/*
-	sets the string to replace in case the var was not assigned
-*/
+    /***[ SetNullString ]******************************************************
+     * @param $str
+     * @param string $varname
+     */
+    /*
+        sets the string to replace in case the var was not assigned
+    */
 
 function SetNullString($str,$varname="") {
 	$this->NULL_STRING[$varname]=$str;
 }
 
-/***[ SetNullBlock ]********************************************************/
-/*
-	sets the string to replace in case the block was not parsed
-*/
+    /***[ SetNullBlock ]*******************************************************
```

Comment:
Isn't this vendored code that we shouldn't mess with?

Reply:
Agree, I must have forgotten to remove the manually included libs before running the PHPDoc fix